test/framework/cluster_proxy.go

```diff
@@ -107,6 +107,8 @@ type ClusterLogCollector interface {
 	// TODO: describe output folder struct
 	CollectMachineLog(ctx context.Context, managementClusterClient client.Client, m *clusterv1.Machine, outputPath string) error
 	CollectMachinePoolLog(ctx context.Context, managementClusterClient client.Client, m *expv1.MachinePool, outputPath string) error
+	// CollectInfrastructureLogs collects log from a kind load balancer
```

Comment:
Is this interface supposed to be provider independent? Then we should probably modify the godoc comment